lib/arjdbc/postgresql/base/pgconn.rb

```diff
@@ -1,8 +1,8 @@
-module ActiveRecord::ConnectionAdapters::PostgreSQL::OID
-  class PGconn # emulate PGconn#unescape_bytea due #652
```

Comment:
@kares Just a heads up that https://github.com/jruby/activerecord-jdbc-adapter/pull/809 makes the change to keep `PG` scoped to where it is referenced.

I'm currently working on making it so that AR-JDBC reuses as much of the core code as possible, but there is definitely more that could be reused if some more methods can be moved around in ActiveRecord itself.